README.md

```diff
@@ -14,98 +14,78 @@ Note: If your ssh public key (`~/.ssh/id_rsa.pub`) is already on the remote mach
 
 ### Inventory
 
-Ansible manages hosts using `inventory.yml` file. Current setup has two group names:
-
-* `sentry`
-* `validator`
-
-Each group may have multiple IPs (hosts). Each Ansible command needs group name to be mentioned. Ansible runs the playbook on mentioned group's machines. Note that if you don't mention group name, Ansible will run playbook on all machines.
+Ansible manages hosts using `inventory.yml` file.
 
 **Setup inventory**
 
-Add sentry nodes IPs/hosts under the `sentry` group and add validator node's IP/host under `validator` group. 
+Add nodes ip's in `inventory.yml` as example
 
 Example:
 
 ```yml
-all:
-  hosts:
-  children:
-    sentry:
-      hosts:
-        xxx.xxx.xx.xx: # <----- Add IP for sentry node
-        xxx.xxx.xx.xx: # <----- Add IP for sentry node
-    validator:
-      hosts:
-        xxx.xxx.xx.xx: # <----- Add IP for validator node
+#add your instance ips here in palce of xx..
+xxx.xxx.xx.xx # <----- Add IP for sentry/validator node
+
 ```
 
 Note: By default the user to login is setup as ubuntu in `group_vars/all` file. If you have a specific user to be logged in with please change the username in this file.
 
 To check if nodes are reachable, run following commands:
 
 ```bash
-# to check if sentry nodes are reachable
-ansible sentry -m ping
-
-# to check if validator nodes are reachable
-ansible validator -m ping
+# to check if nodes are reachable
+ansible -m ping
 ```
 
 ### Networks
 
 There are two networks available:
 
-* `mainnet-v1` (Mainnet v1)
-* `testnet-v4` (Mumbai testnet)
-
-While running Ansible playbook, `network_version` needs to be set.
-
-Similarly for `heimdall_network` following options can be used
-
 * `mainnet` (Mainnet v1)
-* `mumbai`  (Mumbai testnet)
+* `mumbai` (Mumbai testnet)
+
+While running Ansible playbook, `network` `node_type` `heimdall_version` `bor_version` needs to be set.
```

Comment:
We might want to specify where these need to be set. Do you want people to set it in `group_vars/all` just use the `extra-var` field

Reply:
these will be updated as extra var and we have put here the possible values which can be used